src/main/java/net/fabricmc/loom/configuration/CompileConfiguration.java

```diff
@@ -126,6 +132,10 @@ public static void setupConfigurations(Project project) {
 		extendsFrom(Constants.Configurations.LOADER_DEPENDENCIES, Constants.Configurations.MINECRAFT_DEPENDENCIES, project);
 		extendsFrom(Constants.Configurations.MINECRAFT_NAMED, Constants.Configurations.LOADER_DEPENDENCIES, project);
 
+		if (project.getExtensions().getByType(LoomGradleExtension.class).isForge()) {
+			extendsFrom(Constants.Configurations.FORGE_NAMED, Constants.Configurations.LOADER_DEPENDENCIES, project);
```

Comment:
Hm? This seems a bit odd and redundant since they're already on the classpath (`MINECRAFT_NAMED` extends from `LOADER_DEPENDENCIES` above)

Reply:
I did it this way since loom does this for minecraftNamed as well, not sure why they do it this way.

Reply:
Iirc minecraftNamed extends from loaderDeps since the deps were added to the minecraftDependencies before (which was added to minecraftNamed back then). Adding them to forgeNamed shouldn't be necessary at all imo.

Reply:
Iirc minecraftNamed extends from loaderDeps since the deps were added to the minecraftDependencies before (which was added to minecraftNamed back then). Adding them to forgeNamed shouldn't be necessary at all imo.

Reply:
I think this is just easier this way, some stuff in loom depends on `LOADER_DEPENDENCIES`:
```java
Mercury m = new Mercury();
m.setGracefulClasspathChecks(true);

for (File file : project.getConfigurations().getByName(Constants.Configurations.LOADER_DEPENDENCIES).getFiles()) {
    m.getClassPath().add(file.toPath());
}
```

Reply:
This doesn't apply there. `extendsFrom(forgeNamed, loaderDependencies, project)` means that loader deps are included in forge named, not the other way around.

Reply:
Oh, makes sense then